### DISCUSSION TITLE
Some Russian Channel

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#167] [Show and tell]
im not sure if its allowed or not to put those in the list 

https://okko.tv/collection/tvchannels  
18/42 channels if some one know how to proper naming it  ,you will get more

#EXTINF:-1 ,A1
https://okkotv-live.cdnvideo.ru/channel/A1_HD.m3u8
#EXTINF:-1 ,A2
https://okkotv-live.cdnvideo.ru/channel/A2_HD.m3u8
#EXTINF:-1 ,CTC Kids
https://okkotv-live.cdnvideo.ru/channel/CTC_Kids_HD.m3u8
#EXTINF:-1 ,Amedia Hit
https://okkotv-live.cdnvideo.ru/channel/Amedia_Hit_HD.m3u8
#EXTINF:-1,Fox HD
https://okkotv-live.cdnvideo.ru/channel/Fox_HD.m3u8
#EXTINF:-1 ,Fox Life
https://okkotv-live.cdnvideo.ru/channel/Fox_Life_HD.m3u8
#EXTINF:-1 ,VIP Serial
https://okkotv-live.cdnvideo.ru/channel/VIP_Serial_HD.m3u8
#EXTINF:-1 ,VIP Premiere
https://okkotv-live.cdnvideo.ru/channel/VIP_Premiere_HD.m3u8
#EXTINF:-1 ,VIP Megahit
https://okkotv-live.cdnvideo.ru/channel/VIP_Megahit_HD.m3u8
#EXTINF:-1 ,VIP Comedy
https://okkotv-live.cdnvideo.ru/channel/VIP_Comedy_HD.m3u8
#EXTINF:-1 ,TV1000 HD
https://okkotv-live.cdnvideo.ru/channel/TV1000_HD.m3u8
#EXTINF:-1 ,TV1000 Action HD
https://okkotv-live.cdnvideo.ru/channel/TV1000_Action_HD.m3u8
#EXTINF:-1 ,TV1000 Kino
https://okkotv-live.cdnvideo.ru/channel/TV1000_Rus_Kino_HD.m3u8
#EXTINF:-1 ,Sony Sci-Fi
https://okkotv-live.cdnvideo.ru/channel/Sony_SciFi.m3u8
#EXTINF:-1 ,Phoenix
https://okkotv-live.cdnvideo.ru/channel/Phoenix.m3u8
#EXTINF:-1 ,Viasat Nature
https://okkotv-live.cdnvideo.ru/channel/Viasat_Nature_ad_HD.m3u8
#EXTINF:-1 ,Viasat Explore
https://okkotv-live.cdnvideo.ru/channel/Viasat_Explore_HD.m3u8
#EXTINF:-1 ,Viasat History
https://okkotv-live.cdnvideo.ru/channel/Viasat_History_ad_HD.m3u8
#EXTINF:-1,Baby TV
http://okkotv-live.cdnvideo.ru/channel/BabyTV.m3u8

**Comment by Dum4G** [comment]
Just name them the way they are present

**Comment by Sphinxroot** [comment]
not that easy ive tryed but some  are in russian but link naming is english  ,half of them i have guest them

**Comment by NikomaxDj** [comment]
#EXTINF:-1 ,Первый канал HD
https://okkotv-live.cdnvideo.ru/channel/Pervii_Kanal_OTT_HD.m3u8
#EXTINF:-1 ,Россия 1 HD
https://okkotv-live.cdnvideo.ru/channel/Russia1HD.m3u8
#EXTINF:-1 ,Матч! HD
https://okkotv-live.cdnvideo.ru/channel/Match_OTT_HD.m3u8
#EXTINF:-1 ,5 канал
https://okkotv-live.cdnvideo.ru/channel/5_OTT.m3u8
#EXTINF:-1 ,Россия 24
https://okkotv-live.cdnvideo.ru/channel/Russia24.m3u8
#EXTINF:-1 ,Россия Культура
https://okkotv-live.cdnvideo.ru/channel/Russia_K_SD.m3u8
#EXTINF:-1 ,СПАС
https://okkotv-live.cdnvideo.ru/channel/Spas.m3u8
#EXTINF:-1 ,Рен ТВ HD
https://okkotv-live.cdnvideo.ru/channel/Rentv_HD_OTT.m3u8
#EXTINF:-1 ,СТС HD
https://okkotv-live.cdnvideo.ru/channel/CTC_HD_OTT.m3u8
#EXTINF:-1 ,Домашний HD
https://okkotv-live.cdnvideo.ru/channel/Dom_HD_OTT.m3u8
#EXTINF:-1 ,ТВ-3 HD
https://okkotv-live.cdnvideo.ru/channel/TV3_OTT_HD.m3u8
#EXTINF:-1 ,Пятница HD
https://okkotv-live.cdnvideo.ru/channel/Pyatnizza_OTT_HD.m3u8
#EXTINF:-1 ,ТНТ HD
https://okkotv-live.cdnvideo.ru/channel/TNT_OTT_HD.m3u8
#EXTINF:-1 ,Ю
https://okkotv-live.cdnvideo.ru/channel/Yu_OTT.m3u8
#EXTINF:-1 ,Че
https://okkotv-live.cdnvideo.ru/channel/Che_OTT_2.m3u8
#EXTINF:-1 ,СТС Love
https://okkotv-live.cdnvideo.ru/channel/CTC_Love_OTT_2.m3u8
#EXTINF:-1 ,Звезда
https://okkotv-live.cdnvideo.ru/channel/Zvezda_SD.m3u8
#EXTINF:-1 ,Мир
https://okkotv-live.cdnvideo.ru/channel/Mir_OTT.m3u8
#EXTINF:-1 ,МузТВ
https://okkotv-live.cdnvideo.ru/channel/MuzTV.m3u8
#EXTINF:-1 ,Известия
https://okkotv-live.cdnvideo.ru/channel/Izvestiya_HD_2.m3u8
#EXTINF:-1 ,A1
https://okkotv-live.cdnvideo.ru/channel/A1_HD.m3u8
#EXTINF:-1 ,A2
https://okkotv-live.cdnvideo.ru/channel/A2_HD.m3u8
#EXTINF:-1 ,CTC Kids
https://okkotv-live.cdnvideo.ru/channel/CTC_Kids_HD.m3u8
#EXTINF:-1 ,Amedia Hit
https://okkotv-live.cdnvideo.ru/channel/Amedia_Hit_HD.m3u8
#EXTINF:-1,Fox HD
https://okkotv-live.cdnvideo.ru/channel/Fox_HD.m3u8
#EXTINF:-1 ,Fox Life
https://okkotv-live.cdnvideo.ru/channel/Fox_Life_HD.m3u8
#EXTINF:-1 ,VIP Serial
https://okkotv-live.cdnvideo.ru/channel/VIP_Serial_HD.m3u8
#EXTINF:-1 ,VIP Premiere
https://okkotv-live.cdnvideo.ru/channel/VIP_Premiere_HD.m3u8
#EXTINF:-1 ,VIP Megahit
https://okkotv-live.cdnvideo.ru/channel/VIP_Megahit_HD.m3u8
#EXTINF:-1 ,VIP Comedy
https://okkotv-live.cdnvideo.ru/channel/VIP_Comedy_HD.m3u8
#EXTINF:-1 ,TV1000 HD
https://okkotv-live.cdnvideo.ru/channel/TV1000_HD.m3u8
#EXTINF:-1 ,TV1000 Action HD
https://okkotv-live.cdnvideo.ru/channel/TV1000_Action_HD.m3u8
#EXTINF:-1 ,TV1000 Kino
https://okkotv-live.cdnvideo.ru/channel/TV1000_Rus_Kino_HD.m3u8
#EXTINF:-1 ,Sony Sci-Fi
https://okkotv-live.cdnvideo.ru/channel/Sony_SciFi.m3u8
#EXTINF:-1 ,Black
https://okkotv-live.cdnvideo.ru/channel/Sony_Turbo.m3u8
#EXTINF:-1 ,Red
https://okkotv-live.cdnvideo.ru/channel/Sony_ET.m3u8
#EXTINF:-1 ,Феникс Кино
https://okkotv-live.cdnvideo.ru/channel/Phoenix.m3u8
#EXTINF:-1 ,Viasat Nature
https://okkotv-live.cdnvideo.ru/channel/Viasat_Nature_ad_HD.m3u8
#EXTINF:-1 ,Viasat Explore
https://okkotv-live.cdnvideo.ru/channel/Viasat_Explore_HD.m3u8
#EXTINF:-1 ,Viasat History
https://okkotv-live.cdnvideo.ru/channel/Viasat_History_ad_HD.m3u8
#EXTINF:-1 ,Disney
https://okkotv-live.cdnvideo.ru/channel/Disney.m3u8
#EXTINF:-1 ,Мультиландия
https://okkotv-live.cdnvideo.ru/channel/Multilandia_HD.m3u8
#EXTINF:-1 ,National Geographic Wild HD
https://okkotv-live.cdnvideo.ru/channel/NGC_Wild_HD.m3u8
#EXTINF:-1 ,National Geographic HD
https://okkotv-live.cdnvideo.ru/channel/NGC_HD.m3u8
#EXTINF:-1,Baby TV
http://okkotv-live.cdnvideo.ru/channel/BabyTV.m3u8
#EXTINF:-1,Калейдоскоп
https://okkotv-live.cdnvideo.ru/channel/Kaleidoskop.m3u8
#EXTINF:-1,Дикий
http://okkotv-live.cdnvideo.ru/channel/Dikiy.m3u8
#EXTINF:-1,Дикая охота HD
http://okkotv-live.cdnvideo.ru/channel/Dikaya_Ohota_HD.m3u8
#EXTINF:-1,Дикая рыбалка HD
http://okkotv-live.cdnvideo.ru/channel/Dikaya_Rybalka_HD.m3u8
#EXTINF:-1,START Air
https://okkotv-live.cdnvideo.ru/channel/START_Air_HD.m3u8
#EXTINF:-1 ,START World
https://okkotv-live.cdnvideo.ru/channel/START_World_HD.m3u8

**Reply by murik1983** [reply]
спасибо!

**Comment by Delitants** [comment]
Нихера не работает 403

**Reply by Dum4G** [reply]
![image](https://user-images.githubusercontent.com/5430116/154531124-31dcc726-625d-42d5-b924-a8a83aa5e481.png)

Работает в России

**Reply by Delitants** [reply]
С VPS в РФ не работает тоже. Только с домашнего интернета?

**Reply by murik1983** [reply]
в молдове тоже перестали показывать, походу заглушка как с ростелеком

**Reply by NikomaxDj** [reply]
могу сделать переадресацию через роутер keenetic, но трафика не хватит на вас всех.... и потом keenetic может заблокировать домен, а то делал переадресацию рутор, кинозал, звонили с РНК удалить ресурс. Один роутер уже заблокировали...
